parity-crypto/Cargo.toml

```diff
@@ -15,6 +15,9 @@ harness = false
 
 [dependencies]
 tiny-keccak = "1.4"
+eth-secp256k1 = { git = "https://github.com/paritytech/rust-secp256k1" }
+ethereum-types = "0.6.0"
```

Comment:
do we want to use the latest version? (I know we're currently using 0.6 in parity-ethereum, but upgrading that shouldn't be hard)

Reply:
I was wondering if it is doable to avoid 'ethereum-types' dependency. That would probably involve some slight function changes, and more importantly making 'message' a generic type. That is only an idea/question (may even be a bad idea due to possible misuse of message of unmatching len).